cmd/clusterctl/pkg/client/cluster/inventory.go

```diff
@@ -17,7 +17,10 @@ limitations under the License.
 package cluster
 
 import (
+	"time"
+
 	"github.com/pkg/errors"
+	apiextensionsv1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1"
```

Comment:
```suggestion
	apiextensionsv1beta1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1"
```

Also FYI, soon we might switch to CRDv1

Reply:
we are using v1 without RC suffix for everything, are you sure you want to change only this one?
(v1 minimize the impacts of v1beta1 --> v1)

Reply:
we can probably do this later then, if there is a need